docs/user-guide/update-demo/index.md

```diff
@@ -13,11 +13,11 @@ here](https://github.com/kubernetes/kubernetes.github.io/tree/{{page.docsbranch}
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
+This example assumes that you have forked the docs repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
 
 ```shell
-$ cd kubernetes
-$ ./cluster/kube-up.sh
+$ git clone -b {{page.docsbranch}} https://github.com/kubernetes/kubernetes.github.io

```

Comment:
I'm not sure what this include is supposed to accomplish, but we're not hosting versioned docs or anything. We never actually were; GitHub pages doesn't let you host multiple branches. Everything was being cloned to a dummy organization.

You can probably go ahead and take it out.